logging-manager.js

```diff
@@ -41,18 +43,34 @@ const Logger = (module.exports = {
     return this
   },
 
-  checkLogLevel() {
-    fs.readFile('/logging/tracingEndTime', (error, end) => {
-      if (error || !end) {
-        this.logger.level(this.defaultLevel)
-        return
-      }
-      if (parseInt(end) > Date.now()) {
+  async checkLogLevel() {
+    try {
+      const end = await this.getTracingEndTime()
+      if (parseInt(end, 10) > Date.now()) {
         this.logger.level('trace')
       } else {
         this.logger.level(this.defaultLevel)
       }
-    })
+    } catch (err) {
+      console.log(err)
```

Comment:
I am inclined to ignore the error here, `console.log` is a blocking operation in node and is best avoided at run-time.